src/GraphQL/Execution/Nodes/SubscriptionArrayExecutionNode.cs

```diff
@@ -0,0 +1,21 @@
+using GraphQL.Language.AST;
+using GraphQL.Types;
+
+namespace GraphQL.Execution
+{
+    /// <inheritdoc/>
+    public class SubscriptionArrayExecutionNode : ArrayExecutionNode
```

Comment:
These classes were added to the main project because they would be necessary for any subscription implementation.

Reply:
So, maybe move all them with the newly added `Build` method to `GraphQL.SystemReactive` project ?

Reply:
ping

Reply:
well we can but you these are not specific to systemreactive.  i thought you said we were only putting SystemReactive specific implementations in that project?

Reply:
You are right. I'm 50/50. On the one hand, yes. On the other hand, the new classes are only used by this implementation, the only one yet :). We can think of them as implementation details. It is possible that another subscription implementation will create completely different nodes or not create them at all. Who knows?

Reply:
It seemed obvious to me that they would be necessary for any implementation, considering the current design of the project.

Reply:
i don't really care.  i just want to finish these PRs up, finish the documentation up, and release 4.0